extensions/TheShovel/LZ-String.js

```diff
@@ -0,0 +1,275 @@
+(function(Scratch) {
+    'use strict';
+
+    //Code from https://github.com/pieroxy/lz-string/
```

Comment:
copy and paste the `LICENSE` file contents into a comment by after this - https://github.com/pieroxy/lz-string/blob/master/LICENSE